pom.xml

```diff
@@ -632,7 +632,7 @@
         <spark311cdh.version>3.1.1.3.1.7270.0-253</spark311cdh.version>
         <spark312.version>3.1.2</spark312.version>
         <spark313.version>3.1.3-SNAPSHOT</spark313.version>
-        <spark320.version>3.2.0-SNAPSHOT</spark320.version>
+        <spark320.version>3.2.1-SNAPSHOT</spark320.version>
```

Comment:
The Spark320 shim asserts that it sees version 3.2.0.  Don't we need to (temporarily) update the shim to also expect 3.2.1-SNAPSHOT?

Reply:
yes